### DISCUSSION TITLE
Nice to see you again

### DISCUSSION THREAD

**Discussion by creewick** [creewick/MontereyRainmeter#59] [Announcements]
I know there wasn't any news for quite a long time.
I had to move to another country, and I didn't have any Windows PC to continue working on this project, until now.

I plan to re-write this project into a .NET WPF application.
I believe that this way the project will be more expansible and there will be more potential in creating new widgets beyond Rainmeter's limitations.

It's going to be a long way, so stay tuned. I guess, beta versions will be here soon.

Just for now, here's the demo of widgets snapping into a grid
![Запись экрана 2023-04-05 в 23 47 03](https://user-images.githubusercontent.com/32510906/230281971-57b92106-35aa-4840-a14f-b9d7b49351d6.gif)

**Comment by younyokel** [comment]
Nice!

**Reply by WonTheDev** [reply]
looks great!
